### PR TITLE
fix tadd bug

### LINKED ISSUE #222: [PE-D] tadd doesn't show correct messages
When i entered the command as shown in the screenshot, it apparently adds to the members task list, but it doesn't show any success message nor does it show any warning/error message  that I have entered an index "100" that is out of range of the members list.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: Shruthi0310/ped#9

### PR DESCRIPTION
**Pull request by LiuZhi1an**
fixes #222 and add PPP

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/task/TeditCommand.java [hunk 1]
```diff
@@ -31,7 +31,7 @@
     public static final String COMMAND_WORD = "tedit";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the detail of the task identified"
-            + "by the corresponding index number.\n"
+            + " by the corresponding index number.\n"
```
Comment (GWphua) [inline src/main/java/seedu/address/logic/commands/task/TeditCommand.java:34]: Maybe try adding the spacebar at the end in line 33 instead. 

e.g. ": Edits the detail of the task identified "

File: src/test/java/seedu/address/logic/commands/task/TlistCommandTest.java [hunk 2]
```diff
@@ -28,7 +28,7 @@ public void setUp() {
     }
 
     @Test
-    public void execute_tlistIsSuccessfulShown_showsSameList() {
+    public void execute_tlistIsSuccessfullyShown_showsSameList() {
```
Comment (GWphua) [inline src/test/java/seedu/address/logic/commands/task/TlistCommandTest.java:31]: Small English issue: changing to 'execute_tlistSuccessfullyShown_showsSameList()' would be more correct. (take out the Is)

**Review by GWphua — CHANGES_REQUESTED** [review]
Small issue on the wording side, else LGTM.

**Review by tsiyuk — APPROVED** [review]
LGTM

**Review by GWphua — APPROVED** [review]
LGTM

**Pull request merged**